watchmal/model/regression.py

```diff
@@ -18,13 +18,14 @@ def forward(self, x):
         return x
 
 class LinearRegression(nn.Module):
-    def __init__(self, num_inputs, num_outputs):
+    def __init__(self, num_inputs, num_outputs, output):
```

Comment:
Could you please elaborate on why you added the extra "output" parameter here? Thank you.

Reply:
This extra output parameter specifies what data to perform regression on (e.g. position or energy). The configurations for these output types are stored in 'model/regression_alg'. This is used in the regression engine to ensure the loss function compares to the correct output.

Reply:
Is there a reason why this parameter is part of the model class, rather than the engine? It seems it would only be used in the engine and not the model itself.

Reply:
Yes, you're right Nick, it is only used in the engine. I can make the change so that it feeds just into the engine.

Reply:
That would be good if you can, it would make sense for this to be a config parameter of the engine rather than the model I think. We hope to merge this PR quite soon and then continue working on regression, so if you don't have time to make these changes now we can always make another PR with that change in future.

Reply:
I can do that today.

Reply:
looks good, thanks for doing that!

Reply:
Thank you for the clarifaction!